lib/entice/web/client.ex

```diff
@@ -85,11 +87,35 @@ defmodule Entice.Web.Client do
     {:ok, friends}
   end
 
+  @doc "Returns a friend's online status and character name from his account id."
+  def get_status(account_id) do
+    case Entice.Web.Repo.get(Entice.Web.Account, account_id) do
+      nil -> {:error, :no_matching_account}
+      acc ->
+        acc = Entice.Web.Repo.preload(acc, :characters)
+        case Client.Server.get_client(acc.email) do
+          nil ->
+            if length(acc.characters) == 0 do
+              name = "No Character"
+            else
+              name = hd(acc.characters).name
+            end
+            {:ok, :offline, name}
+          id ->
+            attribute = Entity.fetch_attribute(id, Client)
+            assert attribute != :error, "Friend status should never be fetched before entity token."
+            {:ok, client} = attribute
+            player = Player.attributes(client.entity_id)
```

Comment:
this looks suspicious to me... i'd probably do it the same way, but i think this will lead to some refactoring in the future (not THIS exact line of code, but the general way of accessing these player attribute doesnt feel very elegant to me... but nevermind that for now)